cmd/odf/ceph/rbd.go

```diff
@@ -14,17 +12,16 @@ var RbdCmd = &cobra.Command{
 	Short:              "call a 'rbd' CLI command with arbitrary args",
 	DisableFlagParsing: true,
 	Args:               cobra.MinimumNArgs(1),
-	Run: func(cmd *cobra.Command, args []string) {
-		logging.Info("running 'rbd' command with args: %v", args)
-		// verify operator pod is running
-		_, err := k8sutil.WaitForPodToRun(cmd.Context(), root.ClientSets.Kube, root.OperatorNamespace, "app=rook-ceph-operator")
-		if err != nil {
-			logging.Fatal(err)
-		}
+}
 
-		_, err = exec.RunCommandInOperatorPod(cmd.Context(), root.ClientSets, cmd.Use, args, root.OperatorNamespace, root.StorageClusterNamespace, false)
-		if err != nil {
-			logging.Fatal(err)
-		}
```

Comment:
We still need these for running ceph rbd commands

Reply:
@subhamkrai I called a function to verify that the `rook-ceph-operator` pod is running. 
My question is whether we want to support only rbd ls or both `rbd ls` and `rbd ls <pool-name>`. 
What do you think?

Reply:
Let's support what is it in upstream kubectl-rook-ceph is supporting.

Reply:
@subhamkrai 
Upstream kubectl-rook-ceph supports only `rbs ls` and does not support `rbd ls <pool-name>`

Reply:
are you sure @OdedViner I think it will support as `rbd ls <poolname>` is rbd ceph command. can you verify that?